### ISSUE TITLE
Minor manual corrections

### ISSUE THREAD

**Issue by ccfman2004** [Yeriwyn/Salad_Fork#32]
For accented parts you should put [a] or mention when a part is an accented part when calling out printed parts in the manual.

I spent way too much time looking in my printed parts for the A and B tensioners in my primary color parts bin only to double check my printer logs to make sure I did print them only to see that they were accented parts.

Would make things a lot easier.

**Comment by Yeriwyn** [comment]
Good point, I didn't consider that when doing the 1.0 manual. I'm working on completing the new manual for the 1.1 revision, and I'll add callouts for accented parts to it before it's finished.

**Comment by ccfman2004** [comment]
Does the 1.1 revision include magnetic panels?

**Comment by Yeriwyn** [comment]
Not by default. There are new panel retainers, but not magnetic. If someone doesn't add those as a user mod I'll probably do up something a bit later.

Also in reference to your earlier comment about the accent parts, I've just completed doing that in the new manual

**Comment by ccfman2004** [comment]
I have looked at the new PDF manual. It is really good. I wish the Trident manual had all the notes listing all of the lengths of the extrusions for each build size.
